C#/C#.sublime-syntax

```diff
@@ -606,31 +606,41 @@ contexts:
         2: meta.method.cs entity.name.function.cs
         3: meta.method.parameters.cs punctuation.section.parameters.begin.cs
       set: [method_body_transition, method_params]
-    - match: '({{name}}\s*)?((=>)\s*)'
+    - match: (?={{name}})
+      branch_point: method_or_member_variable_name
+      branch:
+        - member_lambda_or_plain
+        - method_name
+      pop: true
+
+  member_lambda_or_plain:
+    - match: ({{name}})\s*(=>)
+      scope: meta.method.cs
       captures:
         1: variable.other.member.cs
-        2: meta.method.cs
-        3: keyword.declaration.function.accessor.get.cs
+        2: keyword.declaration.function.accessor.get.cs
       set:
         - meta_scope: meta.property.cs
         - meta_content_scope: meta.method.cs
         - include: line_of_code_in
-    - match: '({{name}})\s*(?==[^>])'
-      captures:
-        1: variable.other.member.cs
+    - match: '{{name}}'
+      scope: variable.other.member.cs
```

Comment:
Any chance not to break a lambda declaration if the previous attribute decl is not terminated?

```
public class MyClass
{
    bool var

    bool var => return 0;
}
```

![grafik](https://user-images.githubusercontent.com/16542113/138135448-0c09b201-8c15-4367-bb6d-edbe3c333e4c.png)

Reply:
May only one name token be valid in this position and branch?

Found another issue:

![grafik](https://user-images.githubusercontent.com/16542113/138349022-6fdc0f2c-503b-4d00-a30c-3a7181e7f5a1.png)

The closing brace is unscoped due to missing semi-colon.